d3a_interface/api_simulation_config/__init__.py

```diff
@@ -0,0 +1,3 @@
+
+class ApiSimulationConfigException(Exception):
```

Comment:
This one is only used in the `d3a-api-client` so far, right? In that case, I would just put it in that repository.

Reply:
Also on `d3a-web` side to validate the its structure before being served to user https://github.com/gridsingularity/d3a-web/pull/1282

Reply:
I can't seem to find where it is being used in the PR you linked :S

Reply:
Now with this specific link to that line?https://github.com/gridsingularity/d3a-web/pull/1282/files#diff-0708084461f72c4397539a8ca5f0a90cc67058b596fb151ebff2dadb65d72f14R173

Reply:
It seems like you're pointing to a reference to the `validate_api_simulation_config()` method, but I don't see the `ApiSimulationConfigException` there.

Reply:
It will only raise the `ValidationError` in case the returned format isn't in the expected format. Anyways, lets move that discussion to the targeted PR